package.json

```diff
@@ -41,5 +44,21 @@
     "sass": "^1.19.0",
     "sass-loader": "^8.0.0",
     "vue-template-compiler": "^2.6.10"
-  }
+  },
+  "description": "## Running Locally",
+  "main": ".eslintrc.js",
+  "directories": {
+    "test": "tests"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/nativesintech/native-owned-businesses.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
I am not familiar with ISC license. Is it much different from MIT?

Reply:
> The ISC license is functionally equivalent to the BSD 2-Clause and MIT licenses, removing some language that is no longer necessary.

Reply:
https://choosealicense.com/licenses/isc/